.github/workflows/examples.yml

```
@@ -16,79 +16,79 @@ jobs:
+      - name: Save root node_modules to cache
+        uses: actions/cache@v4
+        with:
+          path: node_modules
+          key: v5-package-node-modules-cache-${{ hashFiles('yarn.lock') }}
+      - name: Save root ruby gems to cache
+        uses: actions/cache@v4
+        with:
+          path: vendor/bundle
+          key: package-app-gem-cache-${{ hashFiles('react_on_rails.gemspec') }}-${{ hashFiles('Gemfile.development_dependencies') }}-${{ matrix.versions }}
+      - id: get-sha
+        run: echo "::set-output name=sha::$(git rev-parse HEAD)"
```

actionlint (1.7.4)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)